packages/parser/src/types/envelope.ts

```diff
@@ -0,0 +1,30 @@
+import { apiGatewayEnvelope } from '../envelopes/apigw.js';
```

Comment:
I think these imports can also be converted to `import type ...`, since we are never really using the JS function itself in this file. As far as I can tell it doesn't have an immediate impact on the build output (I've tested both with & without), however since we're using only these as a type I think it's safe to be explicit in the import keyword.